browser/src/map/Map.js

```diff
@@ -1380,6 +1380,77 @@ L.Map = L.Evented.extend({
 		L.DomEvent[onOff](window, 'focus', this._onGotFocus, this);
 	},
 
+	_adjustZoom: function () {
```

Comment:
is that intended part of the PR?

I don't see any relation to the combobox fix

Reply:
the patch was accidentally merged with an older patch.